src/syncData.test.ts

```diff
@@ -0,0 +1,113 @@
+import Redis from 'ioredis'
+import _ from 'lodash/fp.js'
+import {
+  initState as initRedisAndMongoState,
+  numDocs,
+} from 'mongochangestream-testing'
+import { type Db, MongoClient } from 'mongodb'
+import ms from 'ms'
+import { setTimeout } from 'node:timers/promises'
+import { assert, describe, expect, test } from 'vitest'
+
+import * as mongo2crate from './index.js'
+import { type SyncOptions } from './index.js'
+
+const getConns = _.memoize(async () => {
+  // Redis
+  const redis = new Redis({ keyPrefix: 'testing:' })
+  // MongoDB
+  const mongoClient = await MongoClient.connect(
+    process.env.MONGO_CONN as string
```

Comment:
This isn't a blocker, but it would be helpful to document in the README how to set up your environment and run the tests.

Reply:
Done.